### PR TITLE
Update companies.yml

### PR DESCRIPTION
**Pull request by peerst**
TODO: 
- [x] I have read the [contribution guide](CONTRIBUTING.md). 
- [x ] I have checked my spelling and grammar
- [x] I :heart: Erlang/OTP

### CODE REVIEW

**Comment by starbelly** [comment]
Thank you for your outstanding contribution! ❤️ 🚀 🇩🇪

File: src/_data/companies.yml [hunk 1]
```diff
@@ -339,3 +339,11 @@ Loxysoft:
   github: https://github.com/rbkmoney
   location: Moscow, Russia
   description: Erlang is a core of our payment platform serving millions of customers around the world.
+  
+ "Peer Stritzinger GmbH":
```
Comment (starbelly) [inline src/_data/companies.yml]: Oh, I see there is an indentation problem here, could you fix this up?

Comment (peerst) [reply src/_data/companies.yml]: Hopefully fixed, Github online editor had its own ideas how to indent

Comment (starbelly) [reply src/_data/companies.yml]: LGTM!

**Pull request merged**